.changeset/rude-pets-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': minor
+---
+
+Adding TikTok in Social Links
```

Comment:
```suggestion
Adds TikTok social icon
```

We always use present tense verbs in changeset for consistency (if interested, our [changeset conventions](https://contribute.docs.astro.build/docs-for-code-changes/pull-requests/#changesets) can be found here).